service/src/LevelZero.cpp

```diff
@@ -192,7 +193,7 @@ namespace geopm
             //make temp var
             std::vector<zes_freq_handle_t> freq_domain(num_domain);
             check_ze_result(zesDeviceEnumFrequencyDomains(m_devices.at(device_idx).device_handle,
-                                                          &num_domain, freq_domain.data()),
+                            &num_domain, freq_domain.data()),
```

Comment:
Please put this back the way it was.  The indentation indicates that num_domain and freq_domain.data() are parameters to zesDeviceEnumFreqyencyDomains and **not** check_ze_result.